provider/scalewayProvider.js

```diff
@@ -48,7 +48,8 @@ class ScalewayProvider {
     // On serverless info command we do not want log pollution from authentication.
     // This is necessary to use it in an automated environment.
     let hideLog = false;
-    if (this.serverless.configurationInput.service &&
+    if (this.serverless.configurationInput !== undefined &&
```

Comment:
line 51 and line 52 are two different ways to check the same thing, can you be more consistent ?
some reading = https://stackoverflow.com/questions/3390396/how-can-i-check-for-undefined-in-javascript

Reply:
Done :heavy_check_mark: